dpdk_config

```diff
@@ -1,24 +0,0 @@
-CONFIG_RTE_LIBRTE_PMD_BOND=n
-CONFIG_RTE_LIBRTE_PMD_SOFTNIC=n
-CONFIG_RTE_APP_TEST=n
-CONFIG_RTE_TEST_PMD=n
-CONFIG_RTE_MBUF_REFCNT_ATOMIC=n
-CONFIG_RTE_MAX_MEMSEG_LISTS=8192
-CONFIG_RTE_EAL_IGB_UIO=n
-CONFIG_RTE_LIBRTE_KNI=n
-CONFIG_RTE_KNI_KMOD=n
-CONFIG_RTE_LIBRTE_JOBSTATS=n
-CONFIG_RTE_LIBRTE_LPM=n
-CONFIG_RTE_LIBRTE_ACL=n
-CONFIG_RTE_LIBRTE_POWER=n
-CONFIG_RTE_LIBRTE_IP_FRAG=n
-CONFIG_RTE_LIBRTE_DISTRIBUTOR=n
-CONFIG_RTE_LIBRTE_PMD_CRYPTO_SCHEDULER=n
-CONFIG_RTE_LIBRTE_REORDER=n
-CONFIG_RTE_LIBRTE_PORT=n
-CONFIG_RTE_LIBRTE_TABLE=n
-CONFIG_RTE_LIBRTE_PIPELINE=n
-CONFIG_RTE_LIBRTE_FLOW_CLASSIFY=n
-CONFIG_RTE_LIBRTE_BPF=n
-CONFIG_RTE_LIBRTE_EFD=n
-CONFIG_RTE_LIBRTE_MEMBER=n
```

Comment:
Looks like this was truncated, should it be deleted instead?

Reply:
it is deleted. see https://github.com/tchaikov/seastar/tree/wip-dpdk-v3/dpdk_config

Reply:
ok, don't know why I thought otherwise.